website/src/components/views/compareSideBySide/toDownloadLink.ts

```diff
@@ -0,0 +1,21 @@
+import type { DatasetAndVariantData, Id } from '../../../views/View.ts';
+import type { CompareSideBySideStateHandler } from '../../../views/pageStateHandlers/CompareSideBySidePageStateHandler.ts';
+import { toDisplayName } from '../../../views/pageStateHandlers/PageStateHandler.ts';
+
+export function toDownloadLink(pageStateHandler: CompareSideBySideStateHandler) {
+    return ([id, { variantFilter, datasetFilter }]: [Id, DatasetAndVariantData]) => {
+        const filter = pageStateHandler.variantFilterToLapisFilter(datasetFilter, variantFilter);
+        const displayName = toDisplayName(variantFilter);
+        const variantName = displayName.length > 0 ? displayName : `variant ${id + 1}`;
+
+        return {
+            label: `Download accessions of ${variantName}`,
+            filter,
+            downloadFileBasename: `${makeFilenameFriendly(variantName)}_accessions`,
+        };
+    };
+}
+
+export function makeFilenameFriendly(variantName: string) {
```

Comment:
I hope the filename is nice to me ;)
But I'm not sure what this function does. :see_no_evil: Makes it the filename more readable, or better for all filesystems?

Reply:
It makes the variant name filename-friendly. :D I'm not sure what to rename it to, do you have a suggestion?

Reply:
Ah. It is one word :) 
What do you think about: "sanitizeForFilename", or "makeValidFilename"?